Library/Koala/Koala.swift

```diff
@@ -1824,6 +1824,30 @@ public final class Koala {
     }
   }
 
+  public func trackViewedPaymentMethods() {
```

Comment:
I almost want to say we should rebase `master` so that these could be properly placed under `Settings` group. This would require rebasing both this branch and the parent feature branch.
What do you think?